lib/minitest/templates/index.html.erb

```diff
@@ -11,7 +11,9 @@
 <body>
 <div class="container">
     <div class="jumbotron">
-        <h1><%= title %></h1>
+        <h3><small>Generated on <%= DateTime.now.strftime("%b %d, %Y at %H:%M") %></small></h3>
```

Comment:
Are you sure you don't need to specify time zone the reporter should use?  For generic change I'd expect it should work correctly if the reviewer and the reporter creator are in different time zones.

Reply:
Good idea! Done!